build.rs

```diff
@@ -88,3 +88,25 @@ fn android_main() {
     println!("cargo:rustc-link-lib=static=blurdroid");
     println!("cargo:rustc-link-search=native={}", out_dir);
 }
+
+fn find_android_platform(sdk_path: &Path) -> PathBuf {
+    // Sorted android platforms by priority
+    let mut platforms = Vec::new();
+    // Allow configuration using env variable
+    if let Ok(platform) =  env::var("ANDROID_SDK_PLATFORM") {
+        platforms.push(platform);
+    }
+    // Fallback to other platforms
+    for v in (14..26).rev() {
```

Comment:
This will search for android-26 ... android-14, right?
Can you change this to `18..26`? The Android Bluetooth support requires 18 as min version.